### ISSUE TITLE
calibration for tensorRT INT8 in tensorflow failed?

### ISSUE THREAD

**Issue by tilaba** [tensorflow/tensorflow#22633]
when I use the resnetV150_frozen.pb model， I can calibrate the INT8 graph.     However, I try to use another pb model I download from web, I find the INT8 calibration does not work.   I just don't know why?  I doubt the problem may lie in the memory allocation. And when I use data to calibrate the INT8 graph, it always reports the following error.  “ERROR:tensorflow:Not a calib graph. Doesn't seem to contain any calibration nodes.”  please help me!

if f.INT8:
    calibGraph=getINT8CalibGraph(f.batch_size,wsize)
    
    timings,comp,_,mdstats=timeGraph(calibGraph,1,1,dummy_input)
    int8Graph=getINT8InferenceGraph(calibGraph)
    del calibGraph
    timings,comp,valint8,mdstats=timeGraph(int8Graph,f.batch_size,
                                   f.num_loops,dummy_input)
    
  vals=[valnative,valfp32,valfp16,valint8]

def timeGraph(gdef,batch_size=128,num_loops=100,dummy_input=None,timelineName=None):
  tf.logging.info("Starting execution")
  gpu_options = tf.GPUOptions(per_process_gpu_memory_fraction=0.5)
  tf.reset_default_graph()
  g = tf.Graph()
  if dummy_input is None:
    dummy_input = np.random.random_sample((batch_size,3,224,224))
  outlist=[]
  with g.as_default():

    next_element=tf.constant(dummy_input, dtype= tf.float32)
    out = tf.import_graph_def(
      graph_def=gdef,
      input_map={f.input_node:next_element},
      return_elements=[f.output_node]
    )
    out = out[0].outputs[0]
    outlist.append(out)
    
  timings=[]
  
  with tf.Session(graph=g,config=tf.ConfigProto(gpu_options=gpu_options)) as sess:
    run_options = tf.RunOptions(trace_level=tf.RunOptions.FULL_TRACE)
    run_metadata = tf.RunMetadata()
    tf.logging.info("Starting Warmup cycle")
    
    

    num_iters=1
    for i in range(num_loops):
      tstart=time.time()
      for k in range(num_iters):
        val = sess.run(outlist)
      timings.append((time.time()-tstart)/float(num_iters))
      print("iter ",i," ",timings[-1])
    comp=sess.run(tf.reduce_all(tf.equal(val[0],val[0])))
    print("Comparison=",comp)
    sess.close()
    tf.logging.info("Timing loop done!")
    return timings,comp,val[0],None

def getINT8InferenceGraph(calibGraph):
  trt_graph=trt.calib_graph_to_infer_graph(calibGraph)
  return trt_graph

def getINT8CalibGraph(batch_size=128,workspace_size=1<<25):
  trt_graph = trt.create_inference_graph(getResnet50(f.frozen_graph), [ f.output_node],
                                         max_batch_size=batch_size,
                                         max_workspace_size_bytes=workspace_size,
                                         precision_mode="INT8"
                                       )

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by wt-huang** [comment]
@tilaba Try to reduce batch_size to 1 and run the above code. It depends on how different your model is from the one downloaded from the web. Please compare those two models first. Make sure your inference data is representative of the training data.

**Comment by tilaba** [comment]
I tired, but it does not work.

**Comment by tilaba** [comment]
the testing pb model  is attached here . it seems that no node in graph was converted to "TRTengineops"
nodes, that's weird.

**Comment by ankitshah009** [comment]
Any update on how to resolve this error? I am receiving the same error with tensorflow 1-13-1, 
CUDA - 10.0.130 version, NVIDIA driver -  Driver Version: 430.09
